src/components/chartRepo/ChartRepo.tsx

```diff
@@ -143,9 +150,8 @@ const CollapsedList = ({
     }
 
     const handleCollapse = (e) => {
-            e.stopPropagation()
-            toggleCollapse((t) => !t)
-       
+        e.stopPropagation()
+        toggleCollapse((t) => !t)
```

Comment:
why not directly 
```suggestion
        toggleCollapse(!collapsed)
```

Reply:
We can avoid for now, Got the changes from formatting